src/utils.js

```diff
@@ -9,10 +9,40 @@
 
 var fs = require('fs'),
     Q = require('q'),
-    genDecoratorSvgList = require('./client/assets/generate_decorator_svg_list'),
-    ncp = require('ncp'), // Module for copying entire directory
     path = require('path'),
-    requireUncached = require('require-uncached');
+    requireUncached = require('require-uncached'),
+    SVGMapDeffered;
+
+function walkDir (dir, done) {
```

Comment:
I borrowed it from the `generate_decorator_svg.list` and didn't convert it. Will do though